lib/web_ui/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
 
 dev_dependencies:
   archive: 3.1.2
+  file: 6.1.2
```

Comment:
Do we actually need this? I don't see it used anywhere in this PR.

Reply:
Removed. I was tempted to introduce it, but then realized that replacing all the usage of `dart:io` would be scope creep :)